index.rst

```diff
@@ -6,20 +6,423 @@
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
+The safety team is responsible for the the approval of the LOTO procedure, which will be linked on this page once it is affirmed.
+ 
+Start by powering up the instrument cooling electronics, located in the first floor cabinet. 
+It is important to make sure that the electronics are turned on but the outputs are disabled, so no control loops are operating on LATISS until it's under vacuum. 
+These electronics consist of:
+
+-  3 HMP2030 power supplies
+-	1 Keithley 6487 Voltage Source
+-	1 Advantech Uno-1483G-434AE rail controller (not used in the process)
+-	1 Cryocon temperature controller (for CCD sensor)
+
+.. figure:: /_static/power_supply_picometer.png
+   :name: power-supply-picometer
+   :alt: Power supply picometer
+
+   HMP2030 Power Supply (left) and Keithley 6487 Voltage source (right).
+
+.. figure:: /_static/rail_and_cryocon_controller.png
+   :name: rail-and-cryocon-controller
+   :alt: Rail and cryocon controller
+
+   Advantech Rail controller (left) and Cryocon Temperature controller (right).
+
+Setting up the turbo (vacuum) pump
+----------------------------------
+
+The spectrograph needs to work in a good vacuum (~10−7 Torr). For reaching this vacuum, there are two devices involved:
+
+-	Pfeifer Turbopump (to be used from 1 Torr to ~10−6 Torr), complete with vacuum hose, pressure sensor and the sensor’s electronic display
+
+-	Ion pump (to be used from 10−6 Torr onwards)
+
+.. figure:: /_static/turbo_pump_equipment.png
+   :name: turbo-pump-equipment
+   :alt: Turbo pump equipment
+
+   Pfeifer turbopump and accessories.
+
+The Pfeifer turbopump lives in the AuxTel storage container on Calibration Hill. 
+The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container. 
+The cardboard storage box is labeled. 
+The key to the storage container is found in the key box on the second floor (control room floor) of the main building, to the right of the main entrance doors. 
+Mario Rivera also has a copy. 
+
+.. figure:: /_static/auxtel_storage.png
+   :name: auxtel-storage
+   :alt: Auxtel Storage
+
+   Turbo pump supplies located in the storage unit outside of the AuxTel dome.
+
+The ion pump (mounted on LATISS) doesn’t have enough capacity to evacuate the dewar from normal atmospheric pressure to working pressure, so the Pfeifer Turbopump has to be used first. 
+The ion pump should be started only after ~10−6 Torr is reached (as the high pressure could destroy the ion pump).
+
+Connect the turbopump to the dewar
+-----------------------------------
+
+Put the turbopump in a stable position close enough to the dewar position (to be close to the flange connections). 
+An example setup is shown in Figure 5. 
```

Comment:
I think you want to use a reference here, something like,
as shown in :ref:`turbopump-setup`

Reply:
bah, github jacks up the formatting, the name has to be surrounded by these things `

Reply:
👍